src/main/java/ArrayStack.java

```diff
@@ -49,29 +51,44 @@ private void expandCapacity() {
         }
         stack = newStack;
 
-    @Override
-    public void push(T element) {
-
     }
 
-    @Override
```

Comment:
Intentionally removing `@Override`? If so, you've left several on the other methods.

Reply:
not intentional.